src/main/java/uk/gov/di/ipv/cri/common/library/domain/SessionRequest.java

```diff
@@ -22,6 +22,11 @@ public class SessionRequest {
     private SharedClaims sharedClaims;
     private String persistentSessionId;
     private String clientSessionId;
+    private int attemptCount = 0;
+    // TODO: Review the following variables and weather they are needed for passport functionality
+    private String latestDcsResponseResourceId;
+    private String userId;
+    private String govukSigninJourneyId;
```

Comment:
This field (and it's accessor / mutator can be removed )
The `govuk_signin_journey_id` is already assigned to `clientSessionId` as per the previous PR:
( https://github.com/alphagov/di-ipv-cri-lib/pull/42/files )